.github/workflows/pylint.yaml

```diff
@@ -15,10 +15,14 @@ jobs:
         run: python -m pip install --upgrade pip pylint
 
       - name: Setup Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: 3.11
 
+      - name: Remove Dependencies
```

Comment:
We don't need it anymore, see https://github.com/G-Research/DotNetPerfMonitor/pull/6